hallux/targets/github_proposal.py

```diff
@@ -119,10 +119,14 @@ def commit_diff(self) -> bool:
                 commit=commit,
                 side="RIGHT",
                 path=str(compacted.filename),
-                line=compacted.end_line,
-                start_line=compacted.start_line,
+                line=compacted.end_line - 1,
+                start_line=compacted.start_line - 1
+                if compacted.start_line > compacted.end_line
+                else GithubObject.NotSet,
+                start_side="RIGHT" if compacted.start_line > compacted.end_line else GithubObject.NotSet,
             )
-        except GithubException:
+        except BaseException as ex:
```

Comment:
Catch a more specific exception or reraise the exception
```suggestion
                start_side="RIGHT" if compacted.start_line < compacted.end_line else GithubObject.NotSet,
```